lib/matplotlib/tests/test_text.py

```diff
@@ -883,6 +890,7 @@ def test_metrics_cache():
     fig.canvas.draw()
     renderer = fig._get_renderer()
     ys = {}  # mapping of strings to where they were drawn in y with draw_tex.
+    assert renderer in renderer_cache
```

Comment:
Perhaps move this line above the previous one?

Reply:
this is done (but very confusing the way the messages render)